src/steamship/cli/local_server/handler.py

```diff
@@ -0,0 +1,311 @@
+
+    try:
+        request = InvocableRequest.parse_obj(event)
+    except SteamshipError as se:
+        logging.exception(se)
+        return InvocableResponse.from_obj(se)
+    except Exception as ex:
+        logging.exception(ex)
+        return InvocableResponse.error(
+            code=HTTPStatus.INTERNAL_SERVER_ERROR,
+            message="Plugin/App handler was unable to parse inbound request.",
+            exception=ex,
+        )
+
+    if request and request.invocation:
+        error_prefix = (
+            f"[ERROR - {request.invocation.http_verb} {request.invocation.invocation_path}] "
+        )
+    else:
+        error_prefix = "[ERROR - ?VERB ?PATH] "
+
+    if request.invocation.invocation_path == "/__dir__":
+        # Return the DIR result without (1) Constructing invocable_cls or (2) Parsing its config (in the constructor)
+        try:
+            return InvocableResponse(json=invocable_class.__steamship_dir__(invocable_class))
+        except SteamshipError as se:
+            logging.exception(se)
+            return InvocableResponse.from_obj(se)
+        except Exception as ex:
+            logging.exception(ex)
+            return InvocableResponse.error(
+                code=HTTPStatus.INTERNAL_SERVER_ERROR,
+                prefix=error_prefix,
+                message="Unable to initialize package/plugin.",
+                exception=ex,
+            )
+
+    try:
+        invocable = invocable_cls_func()(
+            client=client, config=request.invocation.config, context=invocation_context
+        )
+        # TODO: We don't want to run this every time, but for now we are.
+        logging.info("Running __instance_init__")
+        invocable.instance_init()
+    except SteamshipError as se:
+        logging.exception(se)
+        return InvocableResponse.from_obj(se)
+    except Exception as ex:
+        logging.exception(ex)
+        return InvocableResponse.error(
+            code=HTTPStatus.INTERNAL_SERVER_ERROR,
+            prefix=error_prefix,
+            message="Unable to initialize package/plugin.",
+            exception=ex,
+        )
+
+    if not invocable:
+        return InvocableResponse.error(
+            code=HTTPStatus.INTERNAL_SERVER_ERROR,
+            prefix=error_prefix,
+            message="Unable to construct package/plugin for invocation.",
+        )
+
+    try:
+        response = invocable(request)
+        return InvocableResponse.from_obj(response)
+    except SteamshipError as se:
+        logging.exception(se)
+        se.message = f"{error_prefix}{se.message}"
+        return InvocableResponse.from_obj(se)
+    except Exception as ex:
+        logging.exception(ex)
+        return InvocableResponse.error(
+            code=HTTPStatus.INTERNAL_SERVER_ERROR,
+            prefix=error_prefix,
+            exception=ex,
+        )
+
+
+def handler(bound_internal_handler, event: Dict, _: Dict = None) -> dict:  # noqa: C901
```

Comment:
Is there a way to refactor this method in `lambda_handler` such that so much of this code doesn't have to be copy/pasted here?  Same logic as above comment.